docs/configuration_options.md

```diff
@@ -267,6 +267,7 @@
 | **<a href="#user-content-stress_cmd_complex_prepare" name="stress_cmd_complex_prepare">stress_cmd_complex_prepare</a>**  | cassandra-stress commands.<br>You can specify everything but the -node parameter, which is going to<br>be provided by the test suite infrastructure.<br>multiple commands can passed as a list | N/A | SCT_STRESS_CMD_COMPLEX_PREPARE
 | **<a href="#user-content-prepare_write_stress" name="prepare_write_stress">prepare_write_stress</a>**  | cassandra-stress commands.<br>You can specify everything but the -node parameter, which is going to<br>be provided by the test suite infrastructure.<br>multiple commands can passed as a list | N/A | SCT_PREPARE_WRITE_STRESS
 | **<a href="#user-content-stress_cmd_read_10m" name="stress_cmd_read_10m">stress_cmd_read_10m</a>**  | cassandra-stress commands.<br>You can specify everything but the -node parameter, which is going to<br>be provided by the test suite infrastructure.<br>multiple commands can passed as a list | N/A | SCT_STRESS_CMD_READ_10M
+| **<a href="#user-content-stress_cmd_large_partitions_10m" name="stress_cmd_read_10m">stress_cmd_large_partitions_10m</a>**  | scylla-bench commands.<br>Specify a large-partitions workload using scylla-bench | N/A | SCT_STRESS_CMD_LARGE_PARTITIONS_10M

```

Comment:
Can't we reuse a stress parameter we already have?
We should avoid adding more and more parameters.

Reply:
@roydahan , there is currently no suitable parameter for that.
It's true that the approach of this test is not optimal.
It can be refactor in the future to have a single parameter that contains a dictionary of stress commands, similar to the dictionary concept of full_partition_scan parameter.

Reply:
you could use `stress_cmd_1`, so we won't need to create a new parameter for this specific case, or any other existing parameter that is not "too specific"

Reply:
@fgelcer , it might be too confusing using `scylla-bench` in `stress_cmd_1` where its description is for `cassandra-stress`: 
"""cassandra-stress commands.
                You can specify everything but the -node parameter, which is going to
                be provided by the test suite infrastructure.
                multiple commands can passed as a list"""

Reply:
so fix the `configuration_options.md` to reflect it could be any command, as it is not in use anywhere in SCT codebase

Reply:
@fgelcer , something here doesn't make sence and can't work. this parameter is being used in `test_upgrade_qcl_queries`, but not defined in yaml. i can't use it this way.
```
        InfoEvent(message='Starting c-s read workload for 60m').publish()
        # YAML: stress_cmd_1: cassandra-stress read cl=QUORUM duration=60m -schema 'replication(factor=3)'
        # -mode cql3 native -rate threads=100 -pop seq=1..10000000
        stress_cmd_1 = self._cs_add_node_flag(self.params.get('stress_cmd_1'))
        stress_queue = self.run_stress_thread(stress_cmd=stress_cmd_1)

        InfoEvent(message='Sleeping for 300s to let cassandra-stress start before the upgrade...').publish()
        time.sleep(300)
```
So can you confirmed it's either a bug / unneeded stress / unneeded test/ etc. ?

Reply:
i'm not aware of this test being in use... seems to be an old test, and i don't care to reuse that parameter for a more general purpose, other than creating a new one for every thing

Reply:
fixed